lib/components/PopupList/PopupList.tsx

```diff
@@ -19,21 +17,19 @@ export interface PopupListProps {
 }
 
 export const PopupList = ({ items }: PopupListProps) => {
-  const hasActives = Array.isArray(items) && items.some((item) => item.active);
   return (
-    <div className="inline-flex flex-col items-start rounded-[20px] border-[3px] border-[#767676] bg-white p-6 pr-10">
-      <ul>
+    <div className="inline-flex flex-col items-start rounded bg-white py-6 px-[20px] shadow-border w-[257px]">
+      <ul className="w-full">
         {items.map((item) => {
           const Component = item.type === 'button' ? 'button' : 'a';
 
           return (
-            <li key={item.label} className={hasActives ? 'pl-6' : ''}>
+            <li key={item.label} className={''}>
```

Comment:
I think that empty props/attributes should be removed if they don't serve any purpose

Reply:
Good catch. This is now fixed.